artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -1604,9 +1602,11 @@ public long getDurableScheduledSize() {
 
       for (String lingerSessionId : lingerSessionIds) {
          ServerSession serverSession = server.getSessionByID(lingerSessionId);
-         List<MessageReference> refs = serverSession == null ? null : serverSession.getInTxLingerMessages();
-         if (refs != null && !refs.isEmpty()) {
-            mapReturn.put(serverSession.toManagementString(), refs);
+         if (serverSession != null) {
+            List<MessageReference> refs = serverSession.getInTxLingerMessages();
```

Comment:
I believe there's one semantic change on your change here.

I think it's possible that getInTxLingerMessages() may return null,
so, you changed serverSession != null, but you stopped verifying the actual result. this may cause NPE and tests breaks.

Reply:
your code change may get more complex here than the intended cleanup. I would leave this one, unless you have a creative choise.